packages/examples/grid/src/lib/nextFrame.ts

```diff
@@ -0,0 +1 @@
+﻿export const nextFrame = () => new Promise(ok => requestAnimationFrame(ok))
```

Comment:
What makes this a `lib`?

Reply:
I've gotten rid of the `lib` folder since it bugs you 
1c077df166170c5c351112888b45c373677e94fa